src/tests/gov/nasa/jpf/test/java/io/FileTest.java

```diff
@@ -102,5 +104,18 @@ public void testEquals() {
       assert file.equals(new Object()) == false;
     }
   }
-  
+
+  @Test
+  public void testToURI(){
+    File file = new File("testfile.txt");
+    URI expectedURI = null;
+    try {
+      expectedURI = new File(file.getAbsolutePath()).toURI();
+    } catch (Exception e) {
```

Comment:
Catch a specific subtype of `Exception` so that general failures do not cause the test to pass.

Reply:
Updated unit test